Sources/xcodeproj/PBXFileElement.swift

```diff
@@ -152,7 +152,13 @@ public extension PBXFileElement {
             guard let group = projectObjects.groups.first(where: { $0.value.childrenReferences.contains(reference) }) else { return sourceRoot }
             guard let groupPath = try group.value.fullPath(sourceRoot: sourceRoot) else { return nil }
             guard let filePath = self is PBXVariantGroup ? try baseVariantGroupPath() : path else { return groupPath }
-            return groupPath.appending(RelativePath(filePath))
+            if filePath.hasPrefix("/") {
```

Comment:
I'd use [this constructor](https://github.com/apple/swift-package-manager/blob/master/Sources/Basic/Path.swift#L62) which is intended for this.

Reply:
Done in c11c0d1 👌